include/rppt_tensor_effects_augmentations.h

```diff
@@ -418,6 +418,45 @@ RppStatus rppt_ricap_host(RppPtr_t srcPtr, RpptDescPtr srcDescPtr, RppPtr_t dstP
 RppStatus rppt_ricap_gpu(RppPtr_t srcPtr, RpptDescPtr srcDescPtr, RppPtr_t dstPtr, RpptDescPtr dstDescPtr, Rpp32u *permutedIndicesTensor, RpptROIPtr roiPtrInputCropRegion, RpptRoiType roiType, rppHandle_t rppHandle);
 #endif // GPU_SUPPORT
 
+/*! \brief  Gaussian noise augmentation on HOST backend
+ * \details This function adds gaussian noise to a batch of 4D tensors.
```

Comment:
We say "a batch of 4D" here, but the name of the API is "_3d". That might create some confusion.
So please extend the comment a bit to make it crystal clear.
Open to solutions, but possibly explain something like (A batch of N 3D voxel tensors - each XYZ voxel coordinate supporting 1 or 3 channels)

Reply:
We can this to voxel, which is the suffix we used for flip too, flip_voxel
The gaussian noise HIP API initially had suffix 3d, so i thought i might need to retain 3D for HOST itself and should not change to any other name

Will change the name to voxel